target/linux/ar71xx/image/tp-link.mk

```diff
@@ -384,6 +384,18 @@ define Device/tl-mr13u-v1
 endef
 TARGET_DEVICES += tl-mr10u-v1 tl-mr11u-v1 tl-mr11u-v2 tl-mr12u-v1 tl-mr13u-v1
 
+define Device/tl-mr22u-v1
+  $(Device/tplink-8mlzma)
+  DEVICE_TITLE := TP-LINK TL-MR22U V1
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
```

Comment:
Why did you include `kmod-usb-ledtrig-usbport` here if there is just one LED in your mach-*.C file and it's not used for USB?